### PR TITLE
Add Spanish to site

### LINKED ISSUE #582: i18n on plurals, indicator value suffix and above/below statement in sidepanel
## Describe the task

- [x] update plurals in sidepanel
- [x] update indicator value suffix sidepanel
- [x] fix above/below statement in sidepanel

### LINKED ISSUE #1535: Integrate PR on proper use of i18n

### LINKED ISSUE #1542: Can we add logic to make it say category and threshold when the enumeration is only 1
**Describe the bug**
Currently it says "At or above 1 categories" and "At or above 1 thresholds". It would be better to have "At or above 1 category" and "At or above 1 threshold" in the singular.

**To Reproduce**
1. Go to 'Explore the tool'
2. Click on 'a census tract with at or above 1 category / threshold'
3. Scroll down to 'side panel'

**Expected behavior**
Should be the singular. 

**Screenshots**
Screen Shot 2022-04-08 at 12 28 47 PM

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Additional context**

### PR DESCRIPTION
**Pull request by vim-usds**
# Purpose
The large re-factor to remove i18n anti-patterns is now complete. We also have been able to validate that the Trados tool can translate `en.json` into the format we require so that `es.json` can literally just be dropped in. This PR does a few things:

1. it adds i18n to the ordinal values in the side panel so the `th` in 90th percentile
2. it adds i18n to `... 1 category` and `... 5 categories` in the side panel. This is the way to handle pluralizing
3. adds some format.js linting
4. adds a link to the PR so that anyone can get the new `en.json` file and send it to the translators
5. Spanish feature can be enabled at anytime by adding `.../?flags=sp#...` before the hastag in the URL. This is already on production
 
- closes #1542 
- closes #582 
- closes #1535 

## QA team
The purpose of the QA testing is to test certain parts of the codebase that has changed in particular the side panel. 

The following parts of the side panel are known to not be translated:

Screen Shot 2022-04-08 at 6 24 49 PM

### QA link [here](http://usds-geoplatform-justice40-website.s3-website-us-east-1.amazonaws.com/justice40-tool/1541-da4954/es/?flags=sp#3/33.47/-97.5)

- [x] Do all the indicator values in the side panel look right?
- [x] Does the indicator sub-text (above 90th percentile) also look correct?
- [x] Does the indicator value handle [zero and not available](http://usds-geoplatform-justice40-website.s3-website-us-east-1.amazonaws.com/justice40-tool/1541-da4954/es/?flags=sp#10.86/33.0941/-114.4224) correctly?

## Translation team

The new es.json file is [here](https://github.com/usds/justice40-tool/blob/da495464b3f9c415d06a66a035fa25b20f44c1dd/client/src/intl/en.json) and ready to be downloaded and sent to the translation team.
- [ ] Does the link for the es.json file work

### CODE REVIEW

File: client/src/components/Indicator/Indicator.tsx [hunk 1]
```diff
@@ -78,63 +78,87 @@ export const IndicatorValueSubText = ({value, isAboveThresh, threshold, isPercen
...
-  if (indicator.value !== null) {
-    return indicator.isPercent ?
-        <span>{`%`}</span> :
-        <sup className={styles.indicatorSuperscript}>
-          <span>{getSuperscriptOrdinal(displayStat)}</span>
-        </sup>;
-  } else {
-    return <React.Fragment></React.Fragment>;
-  }
+export const superscriptOrdinal = (indicatorValueWithSuffix:string) => {
```
Comment (esfoobar-usds) [inline client/src/components/Indicator/Indicator.tsx:111]: Really cool! 😎

File: client/src/data/copy/README.md [hunk 2]
```diff
@@ -1,5 +1,21 @@
...
+import {FormattedDate, FormattedMessage, FormattedNumber} from 'gatsby-plugin-intl';
+```
+
+When we are using a place text function with no links, or rich-text formatting, it is suggested to use `defineMessages`.
+
+When we do have to apply rich-text formatting, we can achieve this with `FormattedDate`.
+
+When havin to add Dates or Numbers, we can can use `FormattedDate` and `FormattedNumber` respectively.
+
+More examples can be found [here](https://github.com/formatjs/formatjs/blob/main/packages/react-intl/examples/Messages.tsx)
```
Comment (esfoobar-usds) [inline client/src/data/copy/README.md:21]: ❤️ the docs!

**Review by esfoobar-usds — APPROVED** [review]
🍏

**Comment by vim-usds** [comment]
> > ** 👋 Attention translators!! 👋 ** Copy changes have resulted in a new en.json file. Please download en.json file and send to translators: https://github.com/usds/justice40-tool/blob/da495464b3f9c415d06a66a035fa25b20f44c1dd/client/src/intl/en.json
> 
> FYI @katherinedm-usds, the PR now displays a link to the newest `en.json` if it's changed, so that you can download and send to State. @vim-usds please confirm if what I'm saying is accurate.

@esfoobar-usds and @katherinedm-usds - yes this is correct! TY

**Pull request merged**